jupyterlab_templates/extension.py

```diff
@@ -105,11 +89,16 @@ def load_jupyter_server_extension(nb_server_app):
         nb_server_app (NotebookWebApplication): handle to the Notebook webserver instance.
     """
     web_app = nb_server_app.web_app
-    template_dirs = nb_server_app.config.get("JupyterLabTemplates", {}).get(
-        "template_dirs", []
-    )
+    jupyterlab_templates_config = nb_server_app.config.get("JupyterLabTemplates", {})
+    nb_server_app.log.info("jupyterlab_templates config: %s" % jupyterlab_templates_config)
+
+    template_dirs = jupyterlab_templates_config.get("template_dirs", [])
+    if isinstance(template_dirs, str):
+        template_dirs = template_dirs.split(",")
 
-    if nb_server_app.config.get("JupyterLabTemplates", {}).get("include_default", True):
+    include_default = jupyterlab_templates_config.get("include_default", True)
+    include_default = include_default.lower() == 'true' if isinstance(include_default, str) else include_default
```

Comment:
```suggestion
    include_default = bool(jupyterlab_templates_config.get("include_default", True))
```

It's more readable
```
>>> bool('true')
True
>>> bool('True')
True
>>> bool('false')
True
>>> bool(False)
False
```

Reply:
But when we set this config in the args it's always string. So it will convert the false values incorrectly:
```
bool('false')
Out[6]: True
bool('False')
Out[7]: True
```
Maybe it's better to use the `distutils.util.strtobool`:
```
strtobool('false')
Out[10]: 0
strtobool('False')
Out[11]: 0
strtobool('true')
Out[12]: 1
strtobool('True')
Out[13]: 1
```